src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -98,11 +102,28 @@ class FormPage extends React.Component {
       route: { pageList },
       location,
     } = this.props;
+
     const path = getPreviousPagePath(pageList, form.data, location.pathname);
 
     this.props.router.push(path);
   };
 
+  goToPath = customPath => {
+    const {
+      form,
+      route: { pageList },
+      location,
+    } = this.props;
+
+    const path =
+      customPath &&
+      checkValidPagePath(pageList, this.props.form.data, customPath)
+        ? customPath
+        : getPreviousPagePath(pageList, form.data, location.pathname);
```

Comment:
I added that this will go back one page if the path is invalid, but now that I think about it, maybe it should just do nothing?

Reply:
I'm not sure that doing nothing is a good option, but this sounds like a good question to ask the designer(s) you're working with. Maybe it should go back to the root page the same way an invalid path does for the back button?

Reply:
We'll find out on Monday. Have a great weekend!

Reply:
Ok, spoke with our designer. Going back one page (to a valid destination) would be acceptable if somehow the contact info path were invalid.